workflow.txt

```diff
@@ -1,10 +1,10 @@
+fork on github
 git clone
 git checkout -b my-solution
+git add .
 git commit -m ""
+git push origin my-solution
+git checkout master
 git fetch --all
-git add .
 git pull upstream master
 git merge my-solution
-fork on github
-git checkout master
-git push origin my-solution
```

Comment:
👍 One last step, we need to push our branch to our remote repo.
so while it's not wrong to `git push origin my-solution` after committing our changes above, in order to perform the PR it needs to be pushed again after merging with the fresh version of master.